### PR TITLE
Fixed MessageHandlers in GameComponent

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

**Comment by Pacane** [comment]
LGTM

File: client/web/pokerplanning.dart [hunk 1]
```diff
@@ -41,7 +41,7 @@ class PokerPlanningModule extends Module {
     bind(MessageFactory, toValue: new MessageFactory());
     bind(MessageHandlers,
       toFactory: (MessageFactory messageFactory, kickHandler, resetHandler, gameInformationHandler) {
-        new MessageHandlers(messageFactory, [kickHandler, resetHandler, gameInformationHandler]);
+        return new MessageHandlers(messageFactory, [kickHandler, resetHandler, gameInformationHandler]);
```
Comment (Pacane) [inline client/web/pokerplanning.dart]: whoops. Is that what made staging crash ?

Comment (meriouma) [inline client/web/pokerplanning.dart]: Yep, the messageHandlers variable was null in GameComponent :P

Comment (Pacane) [inline client/web/pokerplanning.dart]: Alright, I'll merge this and test it then.

**Pull request merged**